src/main/scala/org/globalforestwatch/summarystats/gfwpro_dashboard/GfwProDashboardAnalysis.scala

```diff
@@ -36,7 +36,7 @@ object GfwProDashboardAnalysis extends SummaryAnalysis {
     fireAlertRDD: SpatialRDD[Geometry],
     spark: SparkSession,
     kwargs: Map[String, Any]
-  ): Unit = {
```

Comment:
is this change just from a different PR merged in?

Reply:
This (and a bunch of changes shown her) are from my PR adding a new GfwProDashboard test.  This is weird, because my PR and all its changes are already in master.  If you do a diff between origin/feature/Dec23_layer_updates and origin/master, these diffs don't show up, so I think this will be fine.